package.json

```diff
@@ -7,51 +7,51 @@
     "build": "ng build",
     "test": "ng test",
     "test-lib": "ng t --watch=false --browsers=ChromeHeadless --project jdnconvertiblecalendardateadapter",
-    "build-lib": "ng b --project jdnconvertiblecalendardateadapter",
+    "build-lib": "ng b --prod --project jdnconvertiblecalendardateadapter",
```

Comment:
https://angular.io/guide/creating-libraries#publishing-your-library Ivy should not be used when building a library. 

Actually, the compiler shows a warning when not using the `--prod` flag:

> Building Angular Package
> ******************************************************************************
> It is not recommended to publish Ivy libraries to NPM repositories.
> Read more here: https://v9.angular.io/guide/ivy#maintaining-library-compatibility
> ******************************************************************************